docs/concepts/work-pools.md

```diff
@@ -154,9 +154,16 @@ Inspect the work pool:
 
 On success, the command returns the details of the newly created work pool, which can then be used to start agents that poll this pool for work or perform additional configuration of the pool.
 
+
+#### Base Job Configuration <span class="badge beta"></span>
+
+When work pools are created with a worker type, the base job configuration of the work pool can be modified. The base job configuration attributes are specific to a given worker type and allow customization of the default behavior of the worker when executing a flow run. These attributes can then be overridden on a deployment with the `infra_overrides` parameter.
+
+For example, the `ProcessWorker` has a 'Working Directory' field that allows customization of the directory in which flow runs are executed. If a work pool set the default working directory to `/tmp`, then all flow runs executed by the `ProcessWorker` would execute in `/tmp`. If you wanted to override this behavior in a deployment , set the `working_dir` attribute in the `infra_overrides` parameter to a different directory.
+
```

Comment:
It could be nice to include an example of this using `Deployment.build_from_flow()` or something but maybe that's excessive